### ISSUE TITLE
Faux positif avec le dernier mot de passe erroné, ?

### ISSUE THREAD

**Issue by nmeystre** [epfl-si/ATARI#111]
Bonjour,
J'ai remarqué que le champ '

![image](https://github.com/user-attachments/assets/a1bcdde5-906d-443e-85e5-7903e7f0976c)

Dernier mot de passe erroné : 18 septembre 2024 à 09:22 ne correspond pas à la dernière fois où je me suis trompée de mot de passe mais à ma première connexion de la journée.
J'ai constaté cela le 17 et le 18 sept. Pour le 17 j'ai eu un doute (j'ai peut-être fait une erreur de doigt) mais ce main, j'ai été attentive. 9h22 est l'heure où j'ai allumé mon poste (connxion 'windows') avec mon mot de passe correct.
19 septembre: tout est nominal. je continue à faire des tests la semaine prochaine
Cordialement
natalie

**Comment by vigarov** [comment]
J'ai remarque la meme chose, voire plus (des "# de mots passe erronés" trop élevés, des "faux négatifs" = pas de mot de passe erronés alors qu'il y en a un). Au cas ou, de ce que je me souviens de diverses discussions autour de ce champs (meme à l’époque d'idp), je crois qu'il est sourced directement depuis LDAP/l'AD -- je me demande comment c'est fait en backend (coté LDAP/AD) maintenant que l'auth se fait par Entra. Potentiellement à discuter avec Pierre si on veut vraiment fix ce champs

**Comment by ponsfrilus** [comment]
Exact, les valeurs ne sont pas synchronisées entre les serveurs AD. Nous avons refait comme c'était dans idp-exop. S'il est possible d'avoir l'info d'Entra on est preneur !

**Comment by qestoppey** [comment]
C'est ce que j'ai vu dans le code effectivemnt. Pour Entra je ne sais pas ou est cette info, et ce serait que pour le login sur Entra.
